spec/support/paper_trail_helper.rb

```diff
@@ -0,0 +1,13 @@
+# enable PaperTrail for specific tests - you can add a with_versioning test helper method
+def with_versioning
+  was_enabled = PaperTrail.enabled?
+  was_enabled_for_request = PaperTrail.request.enabled?
+  PaperTrail.enabled = true
+  PaperTrail.request.enabled = true
+  begin
+    yield
+  ensure
+    PaperTrail.enabled = was_enabled
+    PaperTrail.request.enabled = was_enabled_for_request
+  end
+end
```

Comment:
we can simplify this using the implict begin in a method
```suggestion
  yield
ensure
  PaperTrail.enabled = was_enabled
  PaperTrail.request.enabled = was_enabled_for_request
end
```